doc/how_to_guides/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -383,6 +383,174 @@ Restrictions for ``MotionSequenceRequest``
+       MoveItConfigsBuilder("moveit_resources_panda")
+       .robot_description(file_path="config/panda.urdf.xacro")
+       .trajectory_execution(file_path="config/gripper_moveit_controllers.yaml")
+       .planning_scene_monitor(
+          publish_robot_description=True, publish_robot_description_semantic=True
+       )
+       .planning_pipelines(
+          pipelines=["pilz_industrial_motion_planner"]
+       )
+       .to_moveit_configs()
+    )
+
+    # Starts Pilz Industrial Motion Planner MoveGroupSequenceAction and MoveGroupSequenceService servers
+    move_group_capabilities = {
+       "capabilities": "pilz_industrial_motion_planner/MoveGroupSequenceAction pilz_industrial_motion_planner/MoveGroupSequenceService"
+    }
+
+The
+:codedir:`pilz_sequence.cpp file <how_to_guides/pilz_industrial_motion_planner/src/pilz_sequence.launch.py>`
+creates 2 target poses that will be reached sequentially.
+
+::
+
+    // ----- Motion Sequence Item 1
+    // Create a MotionSequenceItem
+    moveit_msgs::msg::MotionSequenceItem item1;
+
+    // Set pose blend radius
+    item1.blend_radius = 0.1;
+
+    // MotionSequenceItem configuration
+    item1.req.group_name = PLANNING_GROUP;
+    item1.req.planner_id = "LIN";
+    item1.req.allowed_planning_time = 5;
+    item1.req.max_velocity_scaling_factor = 0.1;
+    item1.req.max_acceleration_scaling_factor = 0.1;
+
+    moveit_msgs::msg::Constraints constraints_item1;
+    moveit_msgs::msg::PositionConstraint pos_constraint_item1;
+    pos_constraint_item1.header.frame_id = "world";
+    pos_constraint_item1.link_name = "panda_hand";
+
+    // Set a constraint pose
+    auto target_pose_item1 = [] {
+       geometry_msgs::msg::PoseStamped msg;
+       msg.header.frame_id = "world";
+       msg.pose.position.x = 0.3;
+       msg.pose.position.y = -0.2;
+       msg.pose.position.z = 0.6;
+       msg.pose.orientation.x = 1.0;
+       msg.pose.orientation.y = 0;
+       msg.pose.orientation.z = 0;
+       msg.pose.orientation.w = 0;
+       return msg;
+    } ();
+    item1.req.goal_constraints.push_back(kinematic_constraints::constructGoalConstraints("panda_link8", target_pose_item1));
+
+The service client needs to be initialized:
+
+::
+
+    // MoveGroupSequence service client
+    using GetMotionSequence = moveit_msgs::srv::GetMotionSequence;
+    auto service_client = node->create_client<GetMotionSequence>("/plan_sequence_path");
+
+    // Verify that the action server is up and running
+    while (!service_client->wait_for_service(std::chrono::seconds(10)))
+    {
+       RCLCPP_WARN(LOGGER, "Waiting for service /plan_sequence_path to be available...");
+    }
+
+Then, the request is created:
+
+::
+
+    // Create request
+    auto service_request = std::make_shared<GetMotionSequence::Request>();
+    service_request->request.items.push_back(item1);
+    service_request->request.items.push_back(item2);
+
+Service call and response. The method ``future.wait_for(timeout_duration)`` waits for the
```

Comment:
This "Service call and response" is not a full sentence, could you reword?

Reply:
I changed the whole sentence to "Once the service call is completed, the method ``future.wait_for(timeout_duration)`` blocks until a specified ``timeout_duration`` has elapsed or the result becomes available, whichever comes first. The return value identifies the state of the result. This is performed every second until the future is ready."